source/MaterialXGenOgsXml/GlslFragmentGenerator.cpp

```diff
@@ -295,14 +410,11 @@ void GlslFragmentGenerator::emitVariableDeclaration(const ShaderPort* variable,
                                                     GenContext& context, ShaderStage& stage,
                                                     bool assignValue) const
 {
-    if (variable->getType() == Type::FILENAME)
-    {
-        emitString("sampler2D " + variable->getVariable(), stage);
-    }
```

Comment:
Don't need this now that the base class correctly handles both uniform and argument samplers.